src/tests/registration/Registration.int.test.tsx

```diff
@@ -0,0 +1,261 @@
+
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.HOME_NUMBER.name}`),
+      {
+        target: { value: TestValue.HomeNumber },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.CELL_NUMBER.name}`),
+      {
+        target: { value: TestValue.CellNumber },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.WORK_NUMBER.name}`),
+      {
+        target: { value: TestValue.WorkNumber },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.EMAIL.name}`),
+      {
+        target: { value: TestValue.Email },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.ADDRESS.name}`),
+      {
+        target: { value: TestValue.Address },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.ParentFavouriteColour },
+      }
+    );
+
+    // children section
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${DefaultFields.FIRST_NAME.name}`),
+      {
+        target: { value: TestValue.ChildFirstName },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${DefaultFields.LAST_NAME.name}`),
+      {
+        target: { value: TestValue.LastName },
+      }
+    );
+
+    const childDobInput = getByLabelText(
+      `${StudentRole.CHILD} ${DefaultFields.DATE_OF_BIRTH.name}`
+    ) as HTMLInputElement;
+    childDobInput.setSelectionRange(0, childDobInput.value.length);
+    userEvent.type(childDobInput, TestValue.ChildDob);
+
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.ChildFavouriteColour },
+      }
+    );
+
+    // guest section
+    fireEvent.click(getByRole("button", { name: "Add member" }));
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${DefaultFields.FIRST_NAME.name}`),
+      {
+        target: { value: TestValue.GuestFirstName },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${DefaultFields.LAST_NAME.name}`),
+      {
+        target: { value: TestValue.LastName },
+      }
+    );
+
+    // session section
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${TEST_SESSION_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.TimeInCanada },
+      }
+    );
+
+    const guestDobInput = getByLabelText(
+      `${StudentRole.GUEST} ${DefaultFields.DATE_OF_BIRTH.name}`
+    ) as HTMLInputElement;
+    guestDobInput.setSelectionRange(0, guestDobInput.value.length);
+    userEvent.type(guestDobInput, TestValue.GuestDob);
+
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.GuestFavouriteColour },
+      }
+    );
```

Comment:
can we move these above L169, so that they show up along with the other guest questions?